### DISCUSSION TITLE
点子2:   抵押EOS获得NFT奖励

### DISCUSSION THREAD

**Discussion by happybig** [famland/website#71] [Ideas]
famland  点子2:   抵押EOS获得NFT奖励

A:  我认为用户进行各种操作应该由famland系统提供CPU  Net等EOS资源

B: 大丰收的资源，都是转账行为，消耗不了多少cpu和net呀
famland，基本上都是合约调用，这个资源消耗级别完全不在一个等级上
这个需要eos开发相应的资源工具就可以解决
因为抵押的成本实在太高，而租用资源目前来看才是更符合我们预期的一种方式

A:   我大概明白了问题的关键点在哪，就在于项目方手上没有EOS。也是随时买资源。
其实也很好解决，开发一个功能: 用户抵押EOS给famland系统，系统支持利息。以这样的方式揽储EOS。

现在EOS收益这么低，有收益的话，EOS飞奔而来

直接给YL最好，YL来源很简单，YL总生成量不变，从沲中分流一部分用来支付EOS质压的收益，也就是说降低一点famland田的收益。这个很好办的。
用YL支付还有一个最大好处就是将YL分散流进持有EOS者手中，这些人对EOS熟悉，最容易转化为famland用户玩家。EOS门外汉转化太费劲

先做一个EOS质压获得YL收益的功能出来添加在网站上
等收到质压的EOS量多了，逐步就用这些EOS资源，支付用户操作资源，逐步替代优化，小碎步，一步一步走

C:  也就是说我们可以通过质押EOS获得的利息，不是去从YL已经分配的部分去抢夺资源，而是从新发行的NFT里面去做文章。这个新发行的NFT又是Famland元宇宙中流通的NFT

D:  完全可以质押eos ,获得一棵柚子树

A:  新构思创建一个全新nft   柚子树
这样这个nft也有价值了，加成价值了，未来还可以延展
不能搞太复杂了，我们做成插拔形式的
以后比较灵活一些
是的， 有了加成属性，就可以定价出售
吸取原始田的经验
未来也可以考虑把原始田也重新生成一下nft
也做成可插拔的模式，原始田本身就具备nft资产属性
或者，弄成柚子挂件，  挂在八宝树上
有八宝树的才能拥有挂件
还赋能了八宝树
八宝树是万树之母
八宝树上，必须得有宝贝，才行
需要把思路开阔一下，我们多用增量思维去延展，不要去触碰现有既得利益人的东西了
慢慢的，弄成了节点， 有节点收益了， 那也不错。 后面可以操作的就太多了
我们就可以定义一个eos树的玩法:
柚子树

最好的是把交易所的柚子全部吸过来， 那柚子不涨价才怪
柚子树给famland加成
实际上eos通过eos树加成的方式，让eos获得了新生

可以就按照资源消耗的比例计算加成系数， 抵押100个eos 获得多少资源，价值多少U， 计算一个系数，铸造一个柚子挂件NFT

是的，  就把eos 引入到了famland生态

现在我们去柚子群里宣传famland ,人家说不支持柚子，不好玩

每抵押100EOS一年，获得一个柚子挂件NFT，抵押100EOS两年，获得三个NFT，一定要尽量长期锁定EOS质压。以后EOS涨价了，质压不到了

**Comment by yanzhiwuli** [comment]
这个建议采纳了，以后抵押EOS 可以获得获得NFT

**Comment by yancanlou** [comment]
这个引流EOS持币玩家的想法不错
